### PR TITLE
Don't invalidate caches except at boot when using disk based caches

### PR DESCRIPTION
**Pull request by rtibbles**
To improve performance on the Raspberry Pi, don't invalidate the cache at download finish, just remove the disk based cache to force a refresh of the content cache at startup.

### CODE REVIEW

**Comment by rtibbles** [comment]
Note: Fixes #4074

File: kalite/updates/views.py [hunk 1]
```diff
@@ -27,7 +27,9 @@ def update_context(request):
 @render_to("updates/update_videos.html")
 def update_videos(request, max_to_show=4):
     context = update_context(request)
-    messages.warning(request, _('For low-powered devices like the Raspberry Pi, please download videos one at a time.'))
+    messages.warning(request, _('For low-powered devices like the Raspberry Pi, please download less than 25 videos at a time.'))
```
Comment (benjaoming) [inline kalite/updates/views.py]: Wondering a bit why this is included, but I previously had this issue opened because the message is very odd anyways, because it's difficult to understand exactly what it means: #3782

Comment (rtibbles) [inline kalite/updates/views.py]: It means.. don't download more than 25 videos at once?

**Comment by benjaoming** [comment]
Great!! Let's aim to include this in Alpha8 and have it for a spin.

File: kalite/updates/views.py (hunk 1 above)
Comment (benjaoming) [inline kalite/updates/views.py]: How is it related to this PR and have you read the issue I've opened about the message? It's more about UI.

File: kalite/caching/__init__.py [hunk 2]
```diff
@@ -65,7 +68,11 @@ def invalidate_all_caches():
     call in here.
     """
     invalidate_inmemory_caches()
-    initialize_content_caches(force=True)
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP:
```
Comment (MCGallaspy) [inline kalite/caching/__init__.py]: I don't really understand _why_ this is necessary, can you help me make the connection in plain English? If we `DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP`, then we should remove all these `*.cache` files, but why?

On the other hand, if that setting is false then we should `initialize_content_caches`, which creates in-memory (as opposed to disk-based) caches?

Comment (rtibbles) [inline kalite/caching/__init__.py]: The underlying assumption here is that if generating in memory caches is too onerous a task to conduct at every system start up, then it is too onerous a task to conduct during server operation.

So we defer the regeneration of these caches to next system startup, by deleting the existing disk based copies of these caches.

This will prompt the caches to be recreated at next system start up, and the disk based copies to be rewritten.

Comment (MCGallaspy) [inline kalite/caching/__init__.py]: Okay, can you explain as much in a comment there? It is perhaps due to my unfamiliarity with this code, but I did not make the connection between deleting these files and not reloading the cache.
